tracer/src/Datadog.Trace/Iast/SensitiveData/TaintedRangeBasedTokenizer.cs

```diff
@@ -17,7 +17,7 @@ namespace Datadog.Trace.Iast.SensitiveData;
 /// </summary>
 internal class TaintedRangeBasedTokenizer : ITokenizer
 {
-    public TaintedRangeBasedTokenizer(TimeSpan timeout)
+    public TaintedRangeBasedTokenizer()
```

Comment:
ooc, why is the timeout not needed anymore?

Reply:
We usually send that parameter to the tokenizers because they use regex. This one does not use them so the parameter was never needed. It was removed just to clean up.